worldedit-core/src/main/java/com/sk89q/worldedit/entity/Entity.java

```diff
@@ -33,7 +33,7 @@
  * instance of an entity, but a {@link BaseEntity} can be created from
  * this entity by calling {@link #getState()}.</p>
  */
-public interface Entity extends Faceted {
+public interface Entity extends Faceted, Locatable {
```

Comment:
I'm unsure if this change is binary-compatible. Usually calling e.g. `Entity#getLocation()` will result in:
```asm
invokevirtual com/sk89q/worldedit/entity/Entity.getLocation:()
```
Which would imply to me that it looks for exactly that method at runtime, and if it's in a superclass it won't be found. But perhaps I am wrong.